testing/scenario_app/lib/main.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:typed_data';
+import 'dart:ui';
+
+import 'src/animated_color_square.dart';
+import 'src/scenario.dart';
+
+Map<String, Scenario> _scenarios = <String, Scenario>{
+  'animated_color_square': AnimatedColorSquareScenario(window),
+};
+
+Scenario _currentScenario = _scenarios['animated_color_square'];
+
+void main() {
+  window
+    ..onPlatformMessage = _handlePlatformMessage
+    ..onBeginFrame = _onBeginFrame
+    ..onDrawFrame = _onDrawFrame
+    ..onMetricsChanged = _onMetricsChanged
+    ..scheduleFrame();
+  final ByteData data = ByteData(1);
+  data.setUint8(0, 1);
+  window.sendPlatformMessage('scenario_status', data, null);
```

Comment:
This is just code comment right? Or does this do anything in this PR

Reply:
This can be used on the embedding side to listen for when the Dart side is ready to get additional instructions.  It's not currently used, I had planned to wire it up at one point and figured it could be done later.